v1/event_format/structured_cloud_event.h

```diff
@@ -0,0 +1,17 @@
+#ifndef CLOUDEVENTCPPSDK_V1_EVENTFORMAT_STRUCTUREDCLOUDEVENT_H_
+#define CLOUDEVENTCPPSDK_V1_EVENTFORMAT_STRUCTUREDCLOUDEVENT_H_
+
+#include "v1/event_format/format.h"
+
+namespace cloudevents {
+namespace format {
+
+struct StructuredCloudEvent {
+    Format format;
```

Comment:
Add comments for each class field to clarify what each function or field is supposed to do. Usually comments always appear in the header files.